tests/NPSP.py

```diff
@@ -309,6 +309,18 @@ def select_lightning_checkbox(self,title):
         locator=npsp_lex_locators['checkbox'].format(title)
         self.selenium.get_webelement(locator).click()
         
+    def select_lightning_table_checkbox(self,title):
+        """"""
+        locator=npsp_lex_locators['table_checkbox'].format(title)
+        self.selenium.get_webelement(locator).click() 
+        
+    def populate_modal_field(self, title, value):
+        locator=npsp_lex_locators['modal_field'].format(title,value)
+        self.selenium.set_focus_to_element(locator)
+        field = self.selenium.get_webelement(locator)
+        field.clear()
+        field.send_keys(value)       
```

Comment:
These keywords don't look NPSP-specific and should end up in the Salesforce library in CumulusCI. It's okay for them to go here temporarily, though.